lib/smart_proxy_dhcp_infoblox/dhcp_infoblox_plugin.rb

```diff
@@ -17,6 +17,8 @@ class Plugin < ::Proxy::Provider
     load_classes ::Proxy::DHCP::Infoblox::PluginConfiguration
     load_dependency_injection_wirings ::Proxy::DHCP::Infoblox::PluginConfiguration
 
+    start_services :unused_ips
```

Comment:
@nofaralfasi Can you move this after the following validates? I think it makes no sense to start the service, when one of the validation fails. Thanks!

Reply:
The order shouldn't really matter since all it does is store the services to be started: https://github.com/theforeman/smart-proxy/blob/e8fe59f83843ca619264e8d4757bc99fcde93fa9/lib/proxy/pluggable.rb#L88-L90

The services are only started once the plugin is fully wired up. If you want to know the details:
* The launcher initializes the plugins using the `PluginInitializer`: https://github.com/theforeman/smart-proxy/blob/e8fe59f83843ca619264e8d4757bc99fcde93fa9/lib/launcher.rb#L129
* The `PluginInitializer` configures all plugins: https://github.com/theforeman/smart-proxy/blob/e8fe59f83843ca619264e8d4757bc99fcde93fa9/lib/proxy/plugin_initializer.rb#L159-L197
* Eventually it configures each `PluginGroup` using the module loader: https://github.com/theforeman/smart-proxy/blob/e8fe59f83843ca619264e8d4757bc99fcde93fa9/lib/proxy/plugin_initializer.rb#L107-L114
* The `DefaultRuntimeConfigurationLoader` wires up the dependencies before starting services: https://github.com/theforeman/smart-proxy/blob/e8fe59f83843ca619264e8d4757bc99fcde93fa9/lib/proxy/plugin_initializer.rb#L238-L239